packages/pilot/src/containers/RecipientsList/index.js

```diff
@@ -94,6 +97,16 @@ const RecipientsList = ({
                 }
                 subtitle={
                   <div>
+                    <div className={style.marginRight}>
+                      <Button
+                        type="button"
+                        icon={<AddIcon width={16} height={16} />}
+                        fill="outline"
+                        onClick={() => { push('./recipients/add') }}
```

Comment:
aqui o ponto é desnecessário, apenas `/recipients/add` é o suficiente

```suggestion
                        onClick={() => { push('/recipients/add') }}
```